src/Resources/ResourceManager/Implementation/CmdletBase/ResourceWithParameterCmdletBase.cs

```diff
@@ -366,18 +376,25 @@ protected Hashtable GetTemplateParameterObject(Hashtable templateParameterObject
                             parameter.Add("reference", dp.Value.Reference);
                         }
 
-                        prameterObject[dp.Key] = parameter;
+                        parameterObject[dp.Key] = parameter;
```

Comment:
I don't see differences between these two, can you please confirm

Reply:
Just correct the naming from prameterObject to p**_a_**rameterObject